src/Interfacer.jl

```diff
@@ -65,6 +67,8 @@ struct CoupledSimulation{
     diagnostics::TD
     callbacks::NTC
     dirs::NTP
+    turbulent_fluxes::TF
+    thermo_params::TP
```

Comment:
We could make this a keyword struct, if that simplifies how it's called. Either way is fine, though.

Reply:
I think I'll just keep it as it is, but I'll keep this in mind!